packages/core-utils/src/types.js

```diff
@@ -20,23 +27,65 @@ export const languageConfigType = PropTypes.shape({
   stopViewer: PropTypes.string
 });
 
+export const mapSymbolsType = PropTypes.arrayOf(
+  PropTypes.shape({
+    dockStrokeColor: PropTypes.string,
+    fillColor: PropTypes.string,
+    maxZoom: PropTypes.number.isRequired,
+    minZoom: PropTypes.number.isRequired,
+    pixels: PropTypes.number,
+    type: PropTypes.string.isRequired
+  }).isRequired
+);
+
 /**
- * Represents the expected configuration of the webapp
+ * Represents the expected configuration of the webapp.
+ *
+ * Note: this is an incomplete type mapping.
```

Comment:
What all is missing? ... maybe list here or the line comment below.

Reply:
There are probably too many things missing and the config itself is sometimes a moving target that may not be worth the time to make a detailed inventory. Ideally this type shouldn't even be used directly in these components that we're extracting. Only parts of the config should be used in each component.